build.gradle

```diff
@@ -116,15 +116,36 @@ repositories {
     maven {
         url "http://tehnut.info/maven"
     }
+	
+	// url of the maven that hosts Enchiridions files
+	maven {
+		url "http://girafi.dk/maven/"
+	}
+	
+	maven {
+		name = "Jared"
+		url = "http://maven.blamejared.com/"
+	}
+	
+	// Baubles
+	maven {
+        name = "JitPack"
+        url = "https://jitpack.io"
+    }
 }
 
 dependencies {
     compile mantleVersion
     compile ticVersion
     compile naturaVersion
-    compile ic2Version
-    compile jeiVersion
+    // compile ic2Version
+    runtime jeiVersion
```

Comment:
Can you explain this?

Reply:
I don't remember my exact reasoning for changing it, but it doesn't really change anything by having it be compile vs runtime. Reverting this change as it shouldn't have been changed in the first place.